canonical-websites/insights.ubuntu.com
# Update insights logo

## Done

Update insights logo with blog logo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Check that the logo does not include the word insights but does include the word blog


## Issue / Card

Fixes [#519](https://github.com/ubuntudesign/web-squad/issues/519)
